Bio/codonalign/codonseq.py

```diff
@@ -262,6 +264,7 @@ def ungap(self, gap=None):
                         rf_table=self.rf_table)
 
     @classmethod
+    """Get condon sequence from sequence data"""
```

Comment:
Missing full stop.

Reply:
Also this is in the wrong place (which is causing the syntax error), it should be below the ``def`` line, not above it.